cmd/installer/cmd/image.go

```diff
@@ -148,6 +148,10 @@ func finalize(platform runtime.Platform, img string) (err error) {
 		if err = ova.CreateOVAFromRAW(name, img, outputArg); err != nil {
 			return err
 		}
+	case "metal":
+		if err = tar(fmt.Sprintf("metal-%s.tar.gz", stdruntime.GOARCH), file, dir); err != nil {
```

Comment:
we need to add this to `.drone.jsonnet` release artifacts

Reply:
Note for myself: we need this to include the board name in the filename if a board name is supplied.